region/brinstar/blue.json

```diff
@@ -1007,6 +1112,66 @@
               ],
               "openEnd": 0
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Geemer - Break the Crumble Blocks",
+                  "notable": false,
+                  "requires": [
+                    "f_ZebesAwake",
+                    {"or": [
+                      "h_canDestroyBombWalls",
+                      {"obstaclesCleared": ["A"]}
+                    ]},
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "h_canFly"
+                    ]}
+                  ],
+                  "note": "Falling down the shaft and breaking the crumble block does not require Morph."
+                },
+                {
+                  "name": "Get Hit By Geemer - Break the Power Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    "f_ZebesAwake",
+                    {"or": [
+                      "h_canDestroyBombWalls",
+                      {"obstaclesCleared": ["A"]}
+                    ]},
+                    "h_canUsePowerBombs",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canShinechargeMovementComplex",
+                        "canFastWalljumpClimb",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 17,
+                          "openEnd": 2
+                        }}
+                      ]}
+                    ]}
+                  ],
+                  "note": "Falling down the shaft and breaking the crumble block does not require Morph."
+                },
+                {
+                  "name": "Get Hit By Geemer - Blocks Already Broken",
```

Comment:
Would it be possible to remove the other strats and only use this one?  Have it find the path where you go down and come back up, or start down and come up.

Reply:
I dont believe that obstacles can be included in cross room strats with the way the logic is currently implemented. It definitely cant be in Map Rando. I will try to work with Maddo to put together a doc about the limitations and some ideas for a new structure soon.